.travis.yml

```diff
@@ -1,12 +1,17 @@
 language: python
 cache: pip
-dist: xenial
+dist: bionic
 services:
   - docker
-addons:
-apt:
-  packages:
-    - python-mysqldb
+before_install:
+  # Add Microsoft SQL Server keys and repository info.
+  - bash -c "wget -qO- https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -"
+  - sudo add-apt-repository "$(wget -qO- https://packages.microsoft.com/config/ubuntu/"$(lsb_release -r -s)"/prod.list)"
+  - sudo apt-get update
+  # Install the ODBC driver for MSSQL, and the MySQL Python bindings requirements.
+  - sudo apt-get install unixodbc-dev python-mysqldb
+  # Install MSSQL Server 2017
+  - sudo ACCEPT_EULA=Y apt-get install msodbcsql17
```

Comment:
This concretely mean two things:
Use the term "Python bindings" instead of "Driver" to describe the "python-myssqldb" package.
Use the same commands in the documentation as the ones that you use here to install the Microsoft SQL Server keys and the ODBC driver.

Reply:
For the `CONTRIBUTING` file, do you know if we actually need `mssql-tools`? There's a package by that name in the apt repository we add for tests, but we don't seem to install it for Travis tests or in the Linux branch of the `CONTRIBUTING` section.

Reply:
I think that we don't, but I can't really say since I don't have a mac. 

If you look at the link below `mssql-tools` seems like a package needed to use bcp and sqlcmd.

https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server?view=sql-server-2017#ubuntu 

I assume that sqlcmd refers to a command-line tool. Below is a link to bcp: 
https://docs.microsoft.com/en-us/sql/tools/bcp-utility?view=sql-server-ver15

Reply:
Ok well this works for now, let's fix it if and when it actually turns out to be broken.